masterlist.yaml

```
@@ -15136,14 +15128,12 @@ plugins:
         condition: 'active("YASH2.esp") and not active("BetterQuestObjectives - YASH Patch.esp")'
     tag: [ Text ]
     clean:
-      - crc: 0x1D471A94
-        util: 'SSEEdit v4.0.4c'
-      - crc: 0x1FC5AB07
-        util: 'SSEEdit v4.0.4c'
+      - crc: 0xDF1DDA2D
+        util: 'SSEEdit v4.1.5f'
```

It looks like you provided cleaning data for the USSEP version, the other version should be kept.

```suggestion
      # USSEP
      - crc: 0xDF1DDA2D
        util: 'SSEEdit v4.1.5f'
      # Non-USSEP
      - crc: 0x1FC5AB07
        util: 'SSEEdit v4.0.4c'
```